web/next/src/components/sidebar/docs.tsx

```
@@ -3,157 +3,142 @@
+  BookOpen,
+  Bot,
+  Brain,
+  Building2,
+  ChevronRight,
+  Cloud,
+  Code,
+  Download,
+  FileText,
+  FolderTree,
+  GitBranch,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)